tmk_core/common/eeconfig.h

```diff
@@ -45,11 +45,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define EECONFIG_HAPTIC (uint32_t *)24
 #define EECONFIG_RGB_MATRIX (uint32_t *)28
-#define EECONFIG_RGB_MATRIX_SPEED (uint8_t *)32
+// Speed & Flags
+#define EECONFIG_RGB_MATRIX_EXTENDED (uint16_t *)32
 // TODO: Combine these into a single word and single block of EEPROM
-#define EECONFIG_KEYMAP_UPPER_BYTE (uint8_t *)33
+#define EECONFIG_KEYMAP_UPPER_BYTE (uint8_t *)34
 // Size of EEPROM being used, other code can refer to this for available EEPROM
-#define EECONFIG_SIZE 34
+#define EECONFIG_SIZE 35
```

Comment:
Since I can't tag it, please increment `EECONFIG_MAGIC_NUMBER` as well.   This way, the shifting EEPROM ranges don't cause weird issues.  (eg, this brute force resets the eeprom to default values)

Reply:
fixed

Reply:
Was fixed in #11046 -- I think we're going to have to decrement it again, for safety's sake.